one-ui/src/main/java/com/marklogic/hub/oneui/auth/MarkLogicAuthenticationManager.java

```diff
@@ -107,10 +109,13 @@ public Authentication authenticate(Authentication authentication) throws Authent
         boolean stagingServerAccessible = false;
         try {
             stagingServerAccessible = dataServicesClient.checkConnection().isConnected();
-        } catch (Exception e) {
+        } catch (Exception ignored) {
```

Comment:
i think we should not ignore the exception after checking the internal implementation of checkConnection().

```suggestion
    DatabaseClient.ConnectionResult connResult;
    try {
      connResult = dataServicesClient.checkConnection();
    } catch (Exception e) {
      throw new InternalAuthenticationServiceException(e.getMessage());
    }

    if (connResult != null && !connResult.isConnected()) {
      if (connResult.getStatusCode() != null && connResult.getStatusCode() == 401) {
        throw new BadCredentialsException(connectionResult.getErrorMessage());
      } else {
        throw new InternalAuthenticationServiceException(connResult.getErrorMessage());
      }
    }
```